spaceship/spec/tunes/b2b_user_spec.rb

```diff
@@ -40,6 +40,20 @@ class B2bUserSpec
         !expect(b2b_user_created.delete)
         expect(b2b_user_created.ds_username).to eq('b2b2@def.com')
       end
+
+      it 'creates correct object to add with explicit input' do
+        b2b_user_created = Spaceship::Tunes::B2bUser.from_username("b2b2@def.com", is_add_type: true)
+        expect(b2b_user_created.add)
+        !expect(b2b_user_created.delete)
```

Comment:
Can we change this line and all other expects to have either `.to eq(true)` or `.to eq(false)` to be more explicit and consistent with the rest of the code base? 😊

Reply:
Ah it was my own rubocop settings - not fastlane's that did this.